Tests/MongoSwiftSyncTests/UnifiedTestRunner/Context.swift

```diff
@@ -1,10 +1,22 @@
+import MongoSwiftSync
+
 /// Context passed around while executing tests.
 class Context {
     /// Path taken to get to the current state.
     var path: [String]
 
-    init(_ path: [String]) {
+    /// Entities created for the test.
+    var entities: EntityMap
```

Comment:
everywhere we pass a context we also pass the entity map, and in order to support some of the newer operations we needed to start passing around the internal client and have some way of recording set fail points too, so it seemed to work nicely to just use `Context` for all of that.